web/src/drafts_overlay_ui.ts

```diff
@@ -299,6 +309,28 @@ export function update_bulk_delete_ui(): void {
     }
 }
 
+export function copy_draft(draft_id: string): void {
+    const draft = draft_model.getDraft(draft_id);
+    if (!draft) {
+        return;
+    }
+
+    const clipboard = new ClipboardJS(
```

Comment:
Hence, you don't have to set up another event handler doing the same thing in `setup_event_handlers()` method. Instead, directly inside the `setup_event_handlers()` method, you could write:
```ts
    new ClipboardJS("#drafts_table .overlay_message_controls .copy-overlay-message", {
        text(trigger): string {
            const draft_id = $(trigger).attr("data-draft-id");
            const draft = drafts.draft_model.getDraft(draft_id);
            return draft?.content ?? "";
        },
    }).on("success", (e) => {
        show_copied_confirmation(e.trigger, {
            show_check_icon: true,
        });
    });
```

Reply:
And include the `data-draft-id` attribute to the HTML of the copy button.

Reply:
Hey @sayamsamal , doing so causing the node tests fail in such a way that I couldn't identify how to fix them.
It will be of great help if you could take a look at the failed test and let me know whether it is fine to do this in this way or let as it is earlier.
Thanks!